backend/sample.env

```diff
@@ -152,3 +152,6 @@ TOOL_REGISTRY_CONFIG_PATH="/data/tool_registry_config"
 
 # Flipt Service
 FLIPT_SERVICE_AVAILABLE=False
+
+# Used by manage.py migrate_to_v2
+SCHEMAS_TO_MIGRATE="_ALL_"
```

Comment:
@chandrasekharan-zipstack This should not be here by default.

Instead, for the one-off v2 migrations pass this directly in command line or some equivalent.